com.github.cameltooling.lsp.eclipse.client.feature/feature.xml

```diff
@@ -2,7 +2,7 @@
 <feature
       id="com.github.cameltooling.lsp.eclipse.client.feature"
       label="Tooling for Apache Camel based on Apache Camel implementation of Language Server Protocol"
-      version="1.0.0.qualifier">
```

Comment:
is that really needed to change? thought it is handled automatically.

Reply:
I don't remember how it is working with tags. Maybe we must even keep it to have the timestamp, will need to investigate

Reply:
I'd say then we need to go like your initial PR